packages/renderer/src/lib/deployments/DeploymentsList.spec.ts

```diff
@@ -33,9 +33,13 @@ import { kubernetesCurrentContextDeployments } from '/@/stores/kubernetes-contex
 import DeploymentsList from './DeploymentsList.svelte';
 
 const kubernetesRegisterGetCurrentContextResourcesMock = vi.fn();
+const getConfigurationValueMock = vi.fn();
```

Comment:
I know this is the way it is already done, but this is not recommended has we lose type safety.

I will provide an example with the existing kubernetesRegisterGetCurrentContextResources.

```ts
beforeAll(() => {
  (window as any).kubernetesRegisterGetCurrentContextResources = vi.fn();
});

// when you need to mock kubernetesRegisterGetCurrentContextResources  you do the following

vi.mocked(window.kubernetesRegisterGetCurrentContextResources).mockResolvedValue([]);
```